RelativeDiscountCalculator.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Price_Calculator_kata
+{
+    internal class RelativeDiscountCalculator : IDiscountCalculator
+    {
+        private double _Discount = 0.0;
+        public double Discount
+        {
+            get => _Discount;
+            set
+            {
+                if (value >= 0 && value < 100.0)
```

Comment:
what if this condition is not matched ? for example a negative value was sent in the contructor ?

Reply:
here nothing changes, should an exception have been thrown?